python_modules/dagster/dagster/_grpc/server.py

```diff
@@ -396,6 +396,15 @@ def _get_repo_for_origin(
             )
         return loaded_repos.definitions_by_name[external_repo_origin.repository_name]
 
+    def ReloadCode(self, _request, _context):
+        self._logger.warn(
+            "Reloading definitions from a code server launched via `dagster api grpc` "
+            "without restarting the process is not currently supported. To enable this functionality, "
+            "launch the code server with the `dagster code-server start` command instead."
+        )
```

Comment:
if i'm only looking at dagit the UI presents as  "successful" right? I see that `ReloadCodeReply` has a `serialized_error` field, should if we use that can we get this message in to dagit?

Reply:
hm does that mean the current unimplemented error doesn't make it to dagit or it does and but treats the location as failed?

Reply:
the UnimplementedError is specifically caught and swallowed as 'expected' in dagit (mostly for backcompat purposes from servers before the call even existed): https://github.com/dagster-io/dagster/blob/master/python_modules/dagster/dagster/_core/host_representation/origin.py#L351-L359)